packages/arb-token-bridge-ui/src/components/TransactionHistory/helpers.ts

```diff
@@ -386,14 +404,31 @@ export async function getUpdatedWithdrawal(
     return tx
   }
 
-  const receipt = await getTxReceipt(tx)
-
-  if (receipt) {
-    const newStatus = getWithdrawalStatusFromReceipt(receipt)
+  const parentChainProvider = getProvider(tx.parentChainId)
+  const childChainProvider = getProvider(tx.childChainId)
```

Comment:
should use `getProviderForChainId` instead

Reply:
Updated